### PR TITLE
AC Automated Tests - Connection wizard stories automation | Master

### PR DESCRIPTION
**Pull request by ViniLopes20**
PR 1 - Ability to connect to AC from Commerce via AC token:
https://issues.liferay.com/browse/LRAC-12443 (Update macros and paths)
https://issues.liferay.com/browse/LRAC-12437 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12438 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12440 (Automation ticket)

PR 2 - Ability to connect to AC from Commerce via AC token:
https://issues.liferay.com/browse/LRAC-12439 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12457 (Update macros and paths)
https://issues.liferay.com/browse/LRAC-12458 (Update macros and paths)

PR 3 - Ability to create new properties:
https://issues.liferay.com/browse/LRAC-12570 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12571 (Automation ticket)

PR 4 - Implement Property pagination during Commerce Connection:
https://issues.liferay.com/browse/LRAC-12583 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12584 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12585 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12586 (Automation ticket)

PR 5 - Ability to create new properties:
https://issues.liferay.com/browse/LRAC-12572 (Automation ticket)

PR 6 - Disconnect AC Workspace:
https://issues.liferay.com/browse/LRAC-12518 (Automation ticket)

PR 7 - Ability to create new properties:
https://issues.liferay.com/browse/LRAC-12574 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12575 (Automation ticket)

PR 8 - Ability to create new properties:
https://issues.liferay.com/browse/LRAC-12576 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12577 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12578 (Automation ticket)

PR 9 - Ability to create new properties:
https://issues.liferay.com/browse/LRAC-12579 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12580 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12581 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12569 (Automation ticket)

PR 10 - Ability to review and change Attributes information:
https://issues.liferay.com/browse/LRAC-12679 (Update macros and paths)

PR 11 - Ability to review and change Attributes information:
https://issues.liferay.com/browse/LRAC-12676 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12677 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12678 (Automation ticket)

PR 12 - Ability to sync All Contacts and Accounts:
https://issues.liferay.com/browse/LRAC-12658 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12659 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12660 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12661 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12662 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12665 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12666 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12667 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12668 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12669 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12670 (Automation ticket)

PR 13 - Ability to sync All Contacts and Accounts:
https://issues.liferay.com/browse/LRAC-12663 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12664 (Automation ticket)

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACDXPSettings.path [hunk 1]
```diff
@@ -97,8 +173,41 @@
...
+	<td>//tr[@data-testid='${key_itemValue}']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>VIEW_LIST_ITEM_ORDER</td>
+	<td>xpath=(//td[@class='table-column-text-start'])[${index}]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>ASSING_MODAL_VIEW_LIST_ITEM_ORDER</td>
```
Comment (christianstokes) [inline portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACDXPSettings.path:189]: Spelling: ASSIGN

File: portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACDXPSettings.path [hunk 2]
```diff
@@ -23,61 +23,137 @@
...
+<tr>
+	<td>ASSING_MODAL_SORT_DIRECTION</td>
+	<td>//div[contains(@class,'fade show')]//*[contains(@class,'order-list-${key_sortDirection}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>ASSING_MODAL_FILTER_AND_ORDER</td>
+	<td>//div[contains(@class,'fade show')]//*[*[normalize-space(text())='Filter and Order']]</td>
+	<td></td>
+</tr>
```
[the remark this answers is not in the document]
Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACDXPSettings.path]: Fixed [here](https://github.com/liferay-ac/liferay-portal/pull/238/commits/2e593cb971771f0d8390534350cbb180adf00208). Thanks for catching this

File: portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACDXPSettings.path (hunk 1 above)
Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACDXPSettings.path]: Fixed [here](https://github.com/liferay-ac/liferay-portal/pull/238/commits/2e593cb971771f0d8390534350cbb180adf00208). Thanks for catching this

**Comment by ViniLopes20** [comment]
Hey guys @joshitagaki @calvinctvo @christianstokes, closing this PR because I sent a [new one](https://github.com/liferay-ac/liferay-portal/pull/256) that covers another story.
cc: @manuferreira

**Pull request closed without merging**